### PR TITLE
Pimp ANSSI up and enable it

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Enable ANSSI profiles and tables.
- Add comments for all requirements of ANSSI
- Selected a few obvious rules for some requirements
- Updated ANSSI references when relevant.

#### Rationale:

  - About 40 requirements (out of 69) have no rule assigned to them. So about 42% requirement coverage.
    - But some requirements are not actionable, like `R16 - Repositories of hardened packages`
- Profile stats:
```
$ python3 ./build-scripts/profile_tool.py stats --benchmark ssg-rhel7-xccdf.xml --profile anssi_nt28_minimal
Profile anssi_nt28_minimal:
* rules:            13
* checks (OVAL):    12	[92% complete]
* fixes (bash):     10	[76% complete]
* fixes (ansible):  10	[76% complete]
* fixes (puppet):   3	[23% complete]
* fixes (anaconda): 3	[23% complete]
* CCEs:             12	[92% complete]

$ python3 ./build-scripts/profile_tool.py stats --benchmark ssg-rhel7-xccdf.xml --profile anssi_nt28_enhanced
Profile anssi_nt28_intermediary:
* rules:            64
* checks (OVAL):    63	[98% complete]
* fixes (bash):     54	[84% complete]
* fixes (ansible):  52	[81% complete]
* fixes (puppet):   3	[4% complete]
* fixes (anaconda): 13	[20% complete]
* CCEs:             61	[95% complete]

$ python3 ./build-scripts/profile_tool.py stats --benchmark ssg-rhel7-xccdf.xml --profile anssi_nt28_high
Profile anssi_nt28_enhanced:
* rules:            74
* checks (OVAL):    73	[98% complete]
* fixes (bash):     59	[79% complete]
* fixes (ansible):  57	[77% complete]
* fixes (puppet):   3	[4% complete]
* fixes (anaconda): 13	[17% complete]
* CCEs:             70	[94% complete]

$ python3 ./build-scripts/profile_tool.py stats --benchmark ssg-rhel7-xccdf.xml --profile anssi_nt28_intermediary
Profile anssi_nt28_high:
* rules:            86
* checks (OVAL):    85	[98% complete]
* fixes (bash):     70	[81% complete]
* fixes (ansible):  65	[75% complete]
* fixes (puppet):   5	[5% complete]
* fixes (anaconda): 15	[17% complete]
* CCEs:             79	[91% complete]
```

The rule count in higher level hardening profiles is summed with lower level hardening profiles.

### CODE REVIEW

File: rhel7/profiles/anssi_nt28_intermediary.profile [hunk 1]
```diff
@@ -11,30 +11,59 @@ extends: anssi_nt28_minimal
...
-    - mount_option_home_nodev
 
-    - sshd_idle_timeout_value=5_minutes
-    - rsyslog_files_ownership
-    - rsyslog_files_groupownership
-    - rsyslog_files_permissions
-    - ensure_logrotate_activated
+    # ==============================================
+    # R14 - Installation of packages reduced to the bare necessities
+    # The choice of packets should lead to an installation as small as possible,
```
Comment (jan-cerny) [inline rhel7/profiles/anssi_nt28_intermediary.profile]: packages?

File: rhel7/profiles/anssi_nt28_enhanced.profile [hunk 2]
```diff
@@ -10,28 +9,112 @@ extends: anssi_nt28_intermediary
...
     - sshd_set_idle_timeout
+    - sshd_idle_timeout_value=5_minutes
 
-    # R35 umask value
+    # ==============================================
+    # R35 - umask value
+    # The system umask must be set to 0027 (by default, any created file can
+    # only be read by the user and his group, and be editable only by his owner).
+    # The umask for users must be set to 0077 (any file created by a user is
+    # readable and editable only by him).
```
Comment (jan-cerny) [inline rhel7/profiles/anssi_nt28_enhanced.profile]: which umask will actually be used in the rules?

File: rhel7/profiles/anssi_nt28_intermediary.profile [hunk 3]
```diff
@@ -194,3 +254,105 @@ selections:
...
+    # of this group should have the right to run sudo.
+
+    # ==============================================
+    # R58 - Sudo configuration guidelines
+    # The following guidelines must be enabled by default:
+    # noexec                apply the NOEXEC tag by default on the commands;
+    # requiretty            require the user to have a tty  login;
+    # use_pty               using a pseudo-tty when a command is executed;
+    # umask = 0027          force umask to a more restrictive mask;
+    # ignore_dot            ignores ithe ‘.’ in $ PATH;
```
Comment (jan-cerny) [inline rhel7/profiles/anssi_nt28_intermediary.profile]: ithe

File: rhel7/profiles/anssi_nt28_enhanced.profile (hunk 2 above)
Comment (yuumasato) [reply rhel7/profiles/anssi_nt28_enhanced.profile]: Good question, the umask value should be set by a variable. I'll check it out later.

Comment (yuumasato) [reply rhel7/profiles/anssi_nt28_enhanced.profile]: I made selection of user `umask` value explicit.

**Pull request merged**